cfy_manager/components/manager/manager.py

```diff
@@ -60,7 +60,7 @@ def _create_sudoers_file_and_disable_sudo_requiretty():
 
 def _get_selinux_state():
     try:
-        return subprocess.check_output('getenforce').rstrip('\n\r')
+        return subprocess.check_output(['/usr/sbin/getenforce']).rstrip('\n\r')
     except OSError:
         logger.warning('SELinux is not installed')
```

Comment:
@tehasdf I recommend also printing the exception.

`logger.warning('SELinux is not installed', exc_info=True)`

Reply:
Or even `logger.warning('SELinux is not installed: {}'.format(str(ex)))`